lib/net/http/responses.rb

```diff
@@ -3,236 +3,601 @@
+  #
+  # Reserved for future use.
+  # See {402 Payment Required}[https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#402].
+  class HTTPPaymentRequired < HTTPClientError
     HAS_BODY = true
   end
-  class HTTPForbidden < HTTPClientError                     # 403
+
+  # Response class for <tt>Forbidden</tt> responses (status code 403).
+  #
+  # The request contained valid data and was understood by the server,
+  # but the server is refusing action.
+  # See {403 Forbidden}[https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#403].
+  class HTTPForbidden < HTTPClientError
     HAS_BODY = true
   end
-  class HTTPNotFound < HTTPClientError                      # 404
+
+  # Response class for <tt>Not Found</tt> responses (status code 404).
+  #
+  # The requested resource could not be found but may be available in the future.
+  # See {404 Not Found}[https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#404].
+  class HTTPNotFound < HTTPClientError
     HAS_BODY = true
   end
-  class HTTPMethodNotAllowed < HTTPClientError              # 405
+
+  # Response class for <tt>Method Not Allowed</tt> responses (status code 405).
+  #
+  # The request method is not supported for the requested resource.
+  # See {405 Method Not Allowed}[https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#405].
+  class HTTPMethodNotAllowed < HTTPClientError
     HAS_BODY = true
   end
-  class HTTPNotAcceptable < HTTPClientError                 # 406
+
+  # Response class for <tt>Not Acceptable</tt> responses (status code 406).
+  #
+  # The requested resource is capable of generating only content
+  # that not acceptable according to the Accept headers sent in the request.
+  # See {406 Not Acceptable}[https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#406].
+  class HTTPNotAcceptable < HTTPClientError
     HAS_BODY = true
   end
-  class HTTPProxyAuthenticationRequired < HTTPClientError   # 407
+
+  # Response class for <tt>Proxy Authentication Required</tt> responses (status code 407).
+  #
+  # The client must first authenticate itself with the proxy.
+  # See {407 Proxy Authentication Required}[https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#407].
+  class HTTPProxyAuthenticationRequired < HTTPClientError
     HAS_BODY = true
   end
-  class HTTPRequestTimeout < HTTPClientError                # 408
+
+  # Response class for <tt>Request Timeout</tt> responses (status code 408).
+  #
+  # The server timed out waiting for the request.
+  # See {408 Request Timeout}[https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#408].
+  class HTTPRequestTimeout < HTTPClientError
     HAS_BODY = true
   end
   HTTPRequestTimeOut = HTTPRequestTimeout
-  class HTTPConflict < HTTPClientError                      # 409
+
+  # Response class for <tt>Conflict</tt> responses (status code 409).
+  #
+  # The request could not be processed because of conflict in the current state of the resource
```

Comment:
You're missing a period for this line.

```suggestion
  # The request could not be processed because of conflict in the current state of the resource.
```

Reply:
Fixed.